src/assets/js/dynamic-addon-cards.js

```diff
@@ -78,27 +93,33 @@
         throw new Error('add-on not found');
       }
 
+      const addon = await response.json();
+
       const {
         name: browserName,
         version: browserVersion,
       } = parsedUserAgent.getBrowser();
       const isFirefox = browserName === 'Firefox';
 
+      const getFirefoxButton = card.querySelector('.GetFirefoxButton');
+
       if (isFirefox) {
         let isIncompatible = false;
         let downloadURL;
         let fileHash;
 
         // Firefox for iOS does not support add-ons.
         if (osName === 'iOS') {
+          // eslint-disable-next-line no-console
```

Comment:
Wondering if it might be easier to turn this off for the file?

Reply:
mmm it was actually turned off for the file :/